CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
+## 1.0.0 (Unreleased)
+
+We are bumping the version of the boundary terraform provider to 1.0.0 and will release new versions of the provider at its own cadence instead of keeping it in lockstep with Boundary.
```

Comment:
```suggestion
We are bumping the version of the Boundary Terraform provider to 1.0.0 and will release new versions of the provider at its own cadence instead of keeping it in lockstep with Boundary.
```